R/Quosure.R

```diff
@@ -0,0 +1,345 @@
+#' Reproducible class with environment and code.
+#'
+#' @slot code (`character`) to reproduce the environment
+#' @slot env (`environment`) environment which content was generated by the evaluation
+#' of the `code` slot.
+setClass(
```

Comment:
?setClass

validity: supplied a validity-checking method for objects from this class. For clearer code, use a separate call to setValidity().
...
prototype: supplies an object with the default data for the slots in this class. A more flexible approach is to write a method for initialize().
...
representation is an argument inherited from S that included both slots and contains, but the use of the latter two arguments is clearer and recommended.